storage/rocksdb/ha_rocksdb.cc

```diff
@@ -1157,8 +1148,43 @@ static MYSQL_SYSVAR_BOOL(pause_background_work, rocksdb_pause_background_work,
 
 static MYSQL_SYSVAR_BOOL(
     enable_ttl, rocksdb_enable_ttl, PLUGIN_VAR_RQCMDARG,
-    "Disable expired ttl records from being dropped during compaction.",
-    nullptr, rocksdb_set_enable_ttl, TRUE);
+    "Enable expired TTL records to be dropped during compaction.", nullptr,
+    nullptr, TRUE);
+
+static MYSQL_SYSVAR_BOOL(
```

Comment:
Have you considered hiding the variables altogether from release builds? This is the usual MySQL practice for debug-only variables, IMHO this would prevent some minor confusion for users seeing them, and the tests need them for debug builds only anyway. The single drawback I can think of is that their use in MTR -master.opt files would require --loose- prefix, introducing uncaught typo risk, but these variables are very unlikely to be used that way.